src/app/api/mcp/[[...mcp]]/route.ts

```
@@ -0,0 +1,530 @@
+      async (args: any) => {
+        await verifyWorkspaceAccess(args.workspaceId);
+        const highestPositionSnapshot = await adminDb
+          .collection("workspaces")
+          .doc(args.workspaceId)
+          .collection("projects")
+          .doc(args.projectId)
+          .collection("tasks")
+          .where("status", "==", args.status)
+          .get();
+        
+        let highestPositionTask = highestPositionSnapshot.docs.map((doc: any) => doc.data()).sort((a: any, b: any) => b.position - a.position)[0];
```

ESLint

[error] 'highestPositionTask' is never reassigned. Use 'const' instead.

(prefer-const)